client/src/modules/authorization/components/AuthInput.jsx

```diff
@@ -59,11 +59,11 @@ class AuthInput extends PureComponent {
 
   setValidationTheme = errorMessageId => {
     const { value } = this.state;
-    const { externalErrorId, formError } = this.props;
+    const { externalErrorId, formError, noValidate } = this.props;
 
     const isValid = !externalErrorId && !errorMessageId;
     const invalidTheme = !isValid || formError;
-    const validTheme = isValid && value && !formError;
+    const validTheme = isValid && value && !formError && !noValidate;
```

Comment:
what does validTheme do?

Reply:
validTheme and invalidTheme decide if and which styles should be applied. These props were here before